frontend/app/.server/routes/mappers/benefit-renewal.state.mapper.ts

```diff
@@ -260,28 +252,28 @@ export class DefaultBenefitRenewalStateMapper implements BenefitRenewalStateMapp
       }),
       communicationPreferences: this.toCommunicationPreferences({
         existingCommunicationPreferences: clientApplication.communicationPreferences,
-        hasEmailChanged,
-        renewedEmail: contactInformation.email,
-        renewedReceiveEmailCommunication: contactInformation.shouldReceiveEmailCommunication,
+        hasEmailChanged: contactInformation?.email ? true : false,
+        renewedEmail: contactInformation?.email,
+        renewedReceiveEmailCommunication: contactInformation?.shouldReceiveEmailCommunication,
       }),
       contactInformation: this.toContactInformation({
         renewedAddressInformation: addressInformation,
         renewedContactInformation: contactInformation,
         existingContactInformation: clientApplication.contactInformation,
-        hasAddressChanged,
-        hasEmailChanged,
-        hasPhoneChanged,
+        hasAddressChanged: addressInformation ? true : false,
+        hasEmailChanged: contactInformation?.email ? true : false,
+        hasPhoneChanged: contactInformation?.phoneNumber ? true : false,
       }),
       dentalBenefits: this.toDentalBenefits({
         existingDentalBenefits: clientApplication.dentalBenefits,
-        hasFederalBenefitsChanged,
-        hasProvincialTerritorialBenefitsChanged,
+        hasFederalBenefitsChanged: dentalBenefits ? true : false,
+        hasProvincialTerritorialBenefitsChanged: dentalBenefits ? true : false,
         renewedDentalBenefits: dentalBenefits,
       }),
       dentalInsurance,
       partnerInformation: this.toPartnerInformation({
         existingPartnerInformation: clientApplication.partnerInformation,
-        hasMaritalStatusChanged,
+        hasMaritalStatusChanged: partnerInformation ? true : false,
```

Comment:
Since it's possible that the applicant changed their marital status and has no partner:

```suggestion
        hasMaritalStatusChanged: maritalStatus ? true : false,
```